src/app/static/src/app/components/Stepper.vue

```diff
@@ -16,11 +16,11 @@
             </template>
         </div>
         <hr/>
-        <div v-if="!ready" class="text-center">
+        <div v-if="loading" class="text-center">
             <loading-spinner size="lg"></loading-spinner>
             <h2 id="loading-message">Loading your data</h2>
         </div>
-        <div v-if="ready" class="content">
+        <div v-if="!loading" class="content">
```

Comment:
As well as showing the loading spinner, on page load the individual steps are all greyed out until data is loaded. I think for the most seamless user experience we should grey out the circles at this point too. Just means updating "ready" to "loading" in 3 places:
https://github.com/mrc-ide/hint/pull/111/files#diff-a9254b1927bebb55fa671243169dccb1L86

https://github.com/mrc-ide/hint/pull/111/files#diff-a9254b1927bebb55fa671243169dccb1L89

https://github.com/mrc-ide/hint/pull/111/files#diff-a9254b1927bebb55fa671243169dccb1L94